Fecha expiración medios off MLA (#734)
mercadopago/devsite-docs: guides/payments/api/other-payment-ways.en.md

>
> Customers have 3 to 5 days to pay, depending on the payment method. After that, you should cancel it.

## Expiration date of cash payment

> The default expiration date for cash payments is 3 days. Optionally, you can change this date by sending the `date_of_expiration` field in the payment creation request. The configured date must be between 1 and 30 days from the issue date.

Reviewer: ```suggestion
The default expiration date for cash payments is 3 days. If you want, you can change this date by sending the `date_of_expiration` field in the payment creation request. The configured date must be between 1 and 30 days from the issue date.
```

[[[
```php
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

$payment->date_of_expiration = "2020-05-30T23:59:59.000-04:00";
```
```node
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

date_of_expiration: "2020-05-30T23:59:59.000-04:00",
```
```java
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

payment.setDateOfExpiration("2020-05-30T23:59:59.000-04:00")
```
```ruby
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

date_of_expiration: "2020-05-30T23:59:59.000-04:00",
```
```csharp
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

payment.DateOfExpiration = DateTime.Parse("2020-05-30T23:59:59.000-04:00");
```
```curl
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

"date_of_expiration": "2020-05-30T23:59:59.000-04:00",
```
]]]

> The deadline for approval of the cash payment is between 1 and 2 working days according to the payment method. Therefore, it is recommended to set the expiration date at least 3 days to ensure that the payment is paid.

Reviewer: ```suggestion
The deadline for approval of the cash payment is between 1 and 2 working days according to the payment method. Therefore, we recommend that you set the due date with at least 3 days to ensure that payment is made.
```

> Check [credit times by payment method](https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/Medios-de-pago-y-acreditaci-n_221) whenever you need to.

Reviewer: ```suggestion
Check [credit times by payment method](https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/Medios-de-pago-y-acreditaci-n_221) when configuring.
```

>
> Customers have 3 to 5 days to pay, depending on the payment method. After that, you should cancel it.

## Expiration date for boleto

> The default expiration date for boleto payments is 3 days. Optionally, you can change this date by sending the `date_of_expiration` field in the payment creation request. The configured date must be between 1 and 30 days from the issue date.

Reviewer: ```suggestion
The default expiration date for boleto payments is 3 days. If you want, you can change this date by sending the `date_of_expiration` field in the payment creation request. The configured date must be between 1 and 30 days from the issue date.
```

[[[
```php
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

$payment->date_of_expiration = "2020-05-30T23:59:59.000-04:00";
```
```node
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

date_of_expiration: "2020-05-30T23:59:59.000-04:00",
```
```java
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

payment.setDateOfExpiration("2020-05-30T23:59:59.000-04:00")
```
```ruby
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

date_of_expiration: "2020-05-30T23:59:59.000-04:00",
```
```csharp
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

payment.DateOfExpiration = DateTime.Parse("2020-05-30T23:59:59.000-04:00");
```
```curl
===
The date uses the ISO 8601 format: yyyy-MM-dd'T'HH:mm:ssz
===

"date_of_expiration": "2020-05-30T23:59:59.000-04:00",
```
]]]

> The deadline for approval of the boleto is up to 48 working hours. Therefore, it is recommended to set the expiration date at least 3 days to ensure that the payment is paid.

Reviewer: ```suggestion
The deadline for approval of the boleto is up to 48 working hours. Therefore, we recommend that you set the due date with at least 3 days to ensure that payment is made.
```
